opm/material/fluidmatrixinteractions/EclMaterialLawManager.hpp

```diff
@@ -117,8 +117,7 @@ class EclMaterialLawManager
                       const std::vector<int>& compressedToCartesianElemIdx)
     {
         // get the number of saturation regions and the number of cells in the deck
-        int ntsFun = deck.getKeyword("TABDIMS").getRecord(0).getItem("NTSFUN").get<int>(0);
-        unsigned numSatRegions = static_cast<unsigned>(ntsFun);
+        const size_t numSatRegions = eclState.runspec().tabdims().getNumSatTables();
```

Comment:
I don't see a Travis complaint, so to be clear: do I need to change anything? If so, what?